src/theory/quantifiers/quantifiers_modules.cpp

```diff
@@ -24,8 +24,9 @@ namespace cvc5 {
 namespace theory {
 namespace quantifiers {
 
-QuantifiersModules::QuantifiersModules()
-    : d_rel_dom(nullptr),
+QuantifiersModules::QuantifiersModules(Env& env)
```

Comment:
This class is purely for initialization of the quantifiers strategy, it doesnt have to be an EnvObj (in another PR, I've added Env& to the initialize function).

Reply:
Makes sense. I've updated this.